### ISSUE TITLE
stacker python 3 support

### ISSUE THREAD

**Issue by noiter** [cloudtools/stacker#465]
When I was running command `stacker --version`, it showed me below error,

`Traceback (most recent call last):
  File "/Users/mz/.pyenv/versions/3.6.2/envs/stacker_venv/bin/stacker", line 3, in <module>
    from stacker.commands import Stacker
  File "/Users/mz/.pyenv/versions/3.6.2/envs/stacker_venv/lib/python3.6/site-packages/stacker/commands/__init__.py", line 1, in <module>
    from .stacker import Stacker  # NOQA
  File "/Users/mz/.pyenv/versions/3.6.2/envs/stacker_venv/lib/python3.6/site-packages/stacker/commands/stacker/__init__.py", line 6, in <module>
    from .diff import Diff
  File "/Users/mz/.pyenv/versions/3.6.2/envs/stacker_venv/lib/python3.6/site-packages/stacker/commands/stacker/diff.py", line 8, in <module>
    from ...actions import diff
  File "/Users/mz/.pyenv/versions/3.6.2/envs/stacker_venv/lib/python3.6/site-packages/stacker/actions/diff.py", line 150
    print "*** No changes to template ***"
 `

This is essentially because I am using Python 3.6.2 and it complains when writing code with `print "something"` rather than `print("something")`. Does it mean stacker has no Python 3 support?

**Comment by phobologic** [comment]
Thanks for the bug report, @noiter - unfortunately stacker does not currently support Python 3.  I'll leave this open though (and modify the subject) in case anyone wants to take this on.

**Comment by phobologic** [comment]
Hey @noiter - just wanted to give you a heads up that I took a stab at this about a week ago, and it got trickier than I expected. It's likely largely due to my total lack of experience with python 3 or converting python 2 apps to python 3.

I've added the `help wanted` tag to this, in case someone who is more experienced with it wants to take a stab.  I'll try to get back to it at some point - but probably not for a while.

Thanks again for pinging us on this!
